examples/sw/benchmarks/coremark/ibex/core_portme.mak

```diff
@@ -0,0 +1,85 @@
+#File : core_portme.mak
+
+OUTFILES = $(OPATH)coremark.dis $(OPATH)coremark.map
```

Comment:
The main coremark makefile uses it similarly (no slash), so I'd prefer to leave it as is. Otherwise when you fail to have a trailing slash you may get slightly confusing behaviour as half of the build system will work and the other half won't.

Reply:
Also, the `$(OPATH)coremark.dis` style is technically better, even if it is a bit confusing: what happens if OPATH is the empty string?